tests/tests_git/test_git_repository.py

```diff
@@ -57,6 +57,13 @@ def test_output(self, mock_subprocess: Any) -> None:
         self.repo.output("echo hello")
         mock_subprocess.assert_called_with("echo hello", cwd=self.repo.dir, shell=True)
 
+    @patch("subprocess.check_output", return_value='927de30 2022-07-12'.encode())
```

Comment:
Add a test for 1) empty output from `check_output`, and 2) multiple lines. In the first case you should get back and empty collection of logs, and in the second multiple items.